timemachine/fe/free_energy.py

```diff
@@ -956,6 +959,7 @@ def get_equilibrated_xvb(xvb: CoordsVelBox, params: NDArray) -> CoordsVelBox:
     fraction_accepted_by_pair_by_iter: List[List[Tuple[int, int]]] = []
 
     for iteration, n_frames_iter in enumerate(batches(md_params.n_frames, n_frames_per_iter), 1):
+        current_step = (iteration - 1) * n_frames_per_iter * md_params.steps_per_frame
```

Comment:
nit: consider moving definition closer to where it is used below

Reply:
Moved in https://github.com/proteneer/timemachine/pull/1220/commits/c49dbe14749ca5d036d478a9337eec46c1cbc415